test/tests/userobjects/internal_side_user_object/internal_side_user_object_two_materials.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  type = FileMesh
+  file = TwoBlockMesh.e
```

Comment:
If you want to wait for your other PR to be merge and then rebase this on top of it, that's fine with me.